dj_cqrs/controller/consumer.py

```diff
@@ -25,11 +26,12 @@ def consume(payload):
         payload.instance_data,
         previous_data=payload.previous_data,
         meta=payload.meta,
+        queue=payload.queue,
     )
 
 
 def route_signal_to_replica_model(
-    signal_type, cqrs_id, instance_data, previous_data=None, meta=None,
+    signal_type, cqrs_id, instance_data, previous_data=None, meta=None, queue=None,
 ):
     """ Routes signal to model method to create/update/delete replica instance.
```

Comment:
It might be interesting to have the new way of documentation syntax like google style even though this function is not in the documentation.

Reply:
As a part of high level refactoring, I agree. But, I guess, that's a story for another PR.